### PR TITLE
Update C++ demos to latest bindings

### PR DESCRIPTION
**Pull request by lupoDharkael**

### CODE REVIEW

File: cpp/cpp_constructors/src/TestClass.hpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+public:
+
+    void _init() {
+        count = 34;
+    }
+
+	int test_method(String s) {
+
+        Godot::print(s);
+		count++;
```
Comment (karroffel) [inline cpp/cpp_constructors/src/TestClass.hpp]: There's some weird formatting going on here, I guess mix of tabes and spaces?

**Comment by karroffel** [comment]
Looking good so far, but maybe you can split the commits in all the different demos you updated? It's easier to review one commit that updates the constructors demo, another one that updates the kinematic body demo and one commit that adds the README.

File: cpp/cpp_constructors/src/TestClass.hpp (hunk 1 above)
Comment (lupoDharkael) [reply cpp/cpp_constructors/src/TestClass.hpp]: Fixed.

**Comment by lupoDharkael** [comment]
Now they are separated commits.

**Comment by aaronfranke** [comment]
@lupoDharkael @karroffel Is there any reason why this isn't in the `master` branch?

**Comment by karroffel** [comment]
I think maybe at the time nativescript 1.1 was not yet released, so it was kept on a separate branch. Now it should probably be the default.

**Pull request merged**